src/opm/parser/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -1211,6 +1215,59 @@ bool FieldProps::tran_active(const std::string& keyword) const {
     return calculator != this->tran.end() && calculator->second.size() > 0;
 }
 
+void FieldProps::apply_numerical_aquifers(const NumericalAquifers& numerical_aquifers) {
+    // TODO: ideally, we should also update the cell_depth, but it is not used for later
```

Comment:
I think we should update all the fieldsprops we can right away, having a stale value in fieldprops can very well lead to trouble at a later stage.

Reply:
Sure.  Let us do it.